viam_cartographer.go

```diff
@@ -433,17 +435,15 @@ func initCartoFacade(ctx context.Context, cartoSvc *CartographerService) error {
 		return err
 	}
 
-	if !cartoSvc.cloudStoryEnabled {
-		err = cf.Start(ctx, cartoSvc.cartoFacadeTimeout)
-		if err != nil {
-			cartoSvc.logger.Errorw("cartofacade start failed", "error", err)
-			termErr := cf.Terminate(ctx, cartoSvc.cartoFacadeTimeout)
-			if termErr != nil {
-				cartoSvc.logger.Errorw("cartofacade terminate failed", "error", termErr)
-				return termErr
-			}
-			return err
```

Comment:
Moving cloud story enabled logic from `viam_cartographer.go` to `carto_facade.cc` so that proper state changes can be set and checked still